ltiauthenticator/lti13/handlers.py

```diff
@@ -50,7 +50,7 @@ async def get(self) -> None:
         target_link_url = f"{protocol}://{self.request.host}"
         self.log.debug(f"Target link url is: {target_link_url}")
         keys = {
-            "title": "JupyterHub",
+            "title": self.authenticator.tool_name,
```

Comment:
Ah, this is possible via BaseHandler: https://github.com/jupyterhub/jupyterhub/blob/049a59f2ed2d5f4f20efd489df6504f403620e02/jupyterhub/handlers/base.py#L166-L168.